src/hooks/useSessionComm.ts

```diff
@@ -25,35 +25,42 @@ export default function useSessionComm(
 
   const allSessionMessages = api.messages.getAllSessionMessages.useQuery({
     sessionId,
-  });
+  }).data;
 
   const addMessageMutation = api.messages.addMessage.useMutation();
 
   api.messages.subscribeToSessionMessages.useSubscription(
     { sessionId, userId },
     {
-      onData: (data) => {
+      onData: (data: Message) => {
         if (data.message) {
           if (data.userId !== userId)
             setChatState((state) => ({
               ...state,
               partnerName: data.userName,
               partnerIsTyping: data.isTyping,
             }));
+
+          if (allSessionMessages)
+            allSessionMessages?.push({
+              ...data,
+              id: data.id!,
+              message: data.message,
+              createdAt: data.createdAt!,
+            });
+          // TODO: Fix autoscroll to bottom
+          const messagesContainer =
```

Comment:
Idt this is a responsibilty of the hook, but rather the messages? So the chat messages should default to the bottom right?

Reply:
Like something liddis, https://stackoverflow.com/questions/18614301/keep-overflow-div-scrolled-to-bottom-unless-user-scrolls-up,
if it takes too much time then dun bother tho

Reply:
yea maybe for another time